### PR TITLE
CitiesSection and CityCard were edited with tailwindcss

### PR DESCRIPTION
**Pull request by min-in-zon**
#17

### CODE REVIEW

File: src/pages/MainPage/CitiesSection.jsx [hunk 1]
```diff
@@ -76,18 +75,21 @@ const CitiesSection = () => {
...
 
   return (
-    <section className="cities">
-      <div className="cities-container">
-        <h2 className="cities-title">Popular Cities</h2>
-        <div className="cities-info">
-          <h3 className="cities-subtitle">
+    <section className="min-h-[48.13rem] pt-28">
+      <div className="container mx-auto">
+        <h2 className="text-2rem font-fourth font-normal">Popular Cities</h2>
```
Comment (adadychanka) [inline src/pages/MainPage/CitiesSection.jsx:80]: `text-2rem`

Is it from overriding in the previous PR?

**Review by adadychanka — CHANGES_REQUESTED** [review]

File: src/components/common/CityCard/CityCard.jsx [hunk 2]
```diff
@@ -1,15 +1,9 @@
...
   return (
-    <div className="card-wrapper">
-      <img
-        className="card-img"
-        src={image}
-        alt={`The picture of ${cityName}`}
-      />
-      <p className="card-city">{cityName}</p>
-      <p className="card-country">{countryName}</p>
+    <div className="rounded-10px flex h-80 w-44 flex-col bg-white/30 font-primary shadow-md">
```
[the remark this answers is not in the document]
Comment (min-in-zon) [reply src/components/common/CityCard/CityCard.jsx:3]: I added borderRadius: {
        '10px': '10px',
      }, in tailwindcfgjs when did hero section. So I use some styles for the following sections, remembering that I had already added these styles in cfg.
      10px is really looks better than xl (12px)

File: src/components/common/CityCard/CityCard.jsx [hunk 3]
```diff
@@ -1,15 +1,9 @@
...
+      <img src={image} alt={`The picture of ${cityName}`} />
+      <p className="ml-1.5 mt-2.5 text-[1.25rem] uppercase">{cityName}</p>
```
[the remark this answers is not in the document]
Comment (min-in-zon) [reply src/components/common/CityCard/CityCard.jsx:5]: My bad

File: src/pages/MainPage/CitiesSection.jsx (hunk 1 above)
Comment (min-in-zon) [reply src/pages/MainPage/CitiesSection.jsx:80]: I used some styles for the following sections, remembering that I had already added these styles in cfg.

**Review by adadychanka — APPROVED** [review]

**Pull request merged**